BlacklightCatalog.lua

```diff
@@ -215,14 +227,14 @@ function IsItemPage()
 			for (let link of aCollection){
 				// A "Back to search" link should be indicative of an item page.
 				if (link.innerText.toLowerCase() == "back to search"){
-					return true;
+					return "True";
```

Comment:
This seems like a strange change to use string instead of booleans since it seems less reliable (or introduces concerns over case matching). Why did we switch?

Reply:
WebView2 can only return strings when evaluating JS, so changing the return values to strings is the easiest way to ensure backwards compatibility with Chromium.